src/main/scala/cromwell/engine/callexecution/CallExecutionActor.scala

```diff
@@ -88,22 +75,35 @@ class CallExecutionActor(backendCall: BackendCall) extends Actor with CromwellAc
     }
   }
 
+  /**
+    * Update the ExecutionHandle
+    */
+  def poll(handle: ExecutionHandle): Future[ExecutionHandle]
+
+  /**
+    * Start the execution. Once the Future resolves, the ExecutionHandle can be used to poll
+    * the state of the execution.
+    */
+  def execute(mode: ExecutionMode): Future[ExecutionHandle]
+  val call: Scope
```

Comment:
As opposed to this which has no default implementation so should probably be a def